stock_picking_package_info/views/product_packaging_view.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright (C) 2016-TODAY LasLabs, Inc. [https://laslabs.com]
```

Comment:
Use new header & remove perpetual copyright. Also, did we create this file? Shouldn't there be another entity copyright here?